index.js

```diff
@@ -1,7 +0,0 @@
-"use strict";
-
-const dist = require("./lib");
-
-module.exports = (...args) => dist.electrodeServer(...args);
-
-Object.assign(module.exports, dist);
```

Comment:
use typescript the way typescript was intended to be used. no weird js shim to overwrite stuff